phy/plot/features.py

```diff
@@ -62,7 +61,7 @@ def features(self, value):
         assert value.ndim == 3
         self.n_spikes, self.n_channels, self.n_features = value.shape
         self._features = value
-        self._non_empty = self.n_spikes > 0
+        self._empty = self.n_spikes == 0
```

Comment:
_empty relate to a parent class.

I expect when I read the code that the self._something relate to the current class.

maybe setEmpty on the parent class, or a property?

(not very important)